src/prism.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable */
+/* http://prismjs.com/download.html?themes=prism-okaidia&languages=markup+clike+javascript+jsx */
```

Comment:
Note [the address where I downloaded this custom Prism build](http://prismjs.com/download.html).

You can see all the options [here](http://prismjs.com/download.html).

Reply:
I wanted to say be careful about adding plugins with this builder. When I tried to add some interesting ones (for example copy to clipboard) It made React puke.  
Getting Uncaught DOMException: Failed to execute 'removeChild' on 'Node' when rerendering component

I think this was because one of the plugins changed the DOM without React knowing. I did not test which one was the culprit since I spent hours trying to troubleshoot already.

Reply:
Nope, I didn't get any linting issues. If you pull down this branch and run it, do you get any linting issues?

Reply:
@jjacode:  I got the eslint errors you described.  I forgot to add the eslint "configuration comment" to `prism.js`

1. download `prism.js` from the [specified url](http://prismjs.com/download.html?themes=prism-okaidia&languages=markup+clike+javascript+jsx)
2. edit this file and place `/* eslint-disable no-eval */` before any javascript

After adding this comment, the errors went away